### PR TITLE
adding readme

### PR DESCRIPTION
**Pull request by melissamohan**
Fixing Exercise #1: Create a README.md #1

### CODE REVIEW

**Comment by snasirca** [comment]
@melissamohan a few things, can you:
- Assign yourself as the "Assignee" on the Issue
- Put me as the Reviewer when the PR is ready, by choosing on the right hand side here
- Give me write access to this repository so that I can accept your PRs

File: README.md [hunk 1]
```diff
@@ -1 +1,56 @@
-# Practice
\ No newline at end of file
+##Project Title
```
Comment (snasirca) [inline README.md]: Can you update the titles of each of the sections to proper markdown? I see the `##` is next to the `Project Title` for example and that prevents it from being rendered as a heading.

**Review by snasirca — CHANGES_REQUESTED** [review]
Made a small suggestion and requesting changes so that you can see the process that takes place when requesting changes 😄

File: README.md (hunk 1 above)
Comment (melissamohan) [reply README.md]: Done, new commit added to resolve this issue

**Review by snasirca — APPROVED** [review]
LGTM 😃. You can merge this into `master`. Try to do a `--no-ff` merge like we do in PackManager. First you'll have to rebase onto `master` your topic branch and then checkout `master` and run a `git merge --no-ff YOUR_TOPIC_BRANCH_NAME`

**Pull request merged**